test/integration/install_simple/controls/server.rb

```diff
@@ -45,6 +45,10 @@
     its("stderr") { should cmp "" }
   end
 
+  describe file("/home/ubuntu/forseti-scripts/run_forseti.sh") do
```

Comment:
Should we try to invoke the cron job or the run_forseti script and ensure that at a minimum the script exits without error?

Reply:
Can you point me to an example I can refer to while adding integration/end-to-end test for `run_forseti.sh`?

Reply:
Added assertions.